.github/workflows/benchmarks.yml

```diff
@@ -1,6 +1,6 @@
 name: Benchmarks
 
-on: [push, pull_request, workflow_call]
+on: workflow_call
 
 jobs:
   ubuntu-build:
```

Comment:
Checkout, build, install, etc. steps should be omitted as their effects are preserved between job runs as I understand?

Reply:
It would be great, but it can't work; the job can be picked by another runner and executed on another machine.